src/main/modules/ardoq/batch/Component.ts

```diff
@@ -0,0 +1,3 @@
+export class Component {
+  constructor(public rootWorkspace: string, public name: string, public typeId: string) {}
```

Comment:
public readonly?

Reply:
actually, if you have a class with no methods what you have is a data structure, maybe it's better just to use an interface defining an object.